jobs/route_registrar/monit

```diff
@@ -0,0 +1,5 @@
+check process route_registrar
```

Comment:
just a question , will there be a dependency between this process and broker process ?

Reply:
No. there won't be any dependency monit process wise. But in health check of route_registrar, it looks for a file placed :https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/blob/master/jobs/service-fabrik-broker/templates/bin/health_check.erb
Also health check looks for 9292 and 9293 port to be up.  So if broker is up and running then only route registration happens successfully. But process runs in wither cases. You see any concern?

Reply:
dont see any concern's since our last conversation.